Aliases/br

```diff
@@ -0,0 +1 @@
+../Formula/brooklyn-client.rb
```

Comment:
At least for now.. 😄 

Is there an accepted convention for naming when the command name itself isn't available, I'll need to run it past the dev list to get consensus on any alternative.